server/base/src/main/javadoc/META-INF/NOTICE

```diff
@@ -1,5 +0,0 @@
-Apache Accumulo
-Copyright 2011-2016 The Apache Software Foundation
```

Comment:
The appropriate LICENSE/NOTICE file are automatically placed in the javadoc jar. The only reason to override is if the javadoc jar needs something different than the source or binary jars. In the case of the server-base module, it doesn't, so these weren't doing anything helpful. It was just extra. Removing them makes it behave just like the gc or start jars, which don't have anything special (special == bundled external code) either.

The only modules which require something special in their jars are core (for bloom filter stuffs) and monitor (for jquery/flot).